doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -1225,6 +1225,12 @@ Other libraries
 
   * Added the :c:func:`date_time_now_local` function to the API.
 
+* :ref:`dk_buttons_and_leds_readme` library:
+
+  * Added an experimental no interrupts mode for button handling.
```

Comment:
Somehow, the name of this mode "no interrupts" does not stand out from the text. Still, I don't really know whether we can highlight in any way. @annwoj, any suggestions?

Reply:
That was on my mind as well earlier, but using any typographical conventions here doesn't really make much sense. We could capitalize it but it won't align with other similar occurrences in the ncs documentation, so I don't really have any good solution for this :(. Figured that the best way not to confuse anyone is leaving is as is. 
Now I just thought of "no-interrupts mode" to highlight this part, and although there are hyphenated usages of this out there I'm not sure how justified it is in terms of punctuation.